Gruntfile.js

```diff
@@ -28,6 +29,19 @@ module.exports = function(grunt) {
     pkg: grunt.file.readJSON('package.json'),
 
     copy: {
+      'types': {
+        files: [
+          {
+            expand: true,
+            cwd: COURAGE_TYPES,
+            src: [
+              '**/*.d.ts'
+            ],
+            dest: `types/generated/${COURAGE_TYPES}`
+          }
```

Comment:
why need to copy courage types to top level.

Reply:
courage is not available as an installable module so all necessary courage types are published in a folder underneath the widget's type exports

Reply:
For my education, cannot we generate a single `index.d.ts` which includes all (like `backbone/index.d.ts`. although it may not be generated by hand crafted)